contracts/TranscodeJobs.sol

```diff
@@ -63,6 +66,8 @@ library TranscodeJobs {
         self.jobs[self.numJobs].transcoderAddress = _electedTranscoder;
         self.numJobs++;
 
+        NewJob(_electedTranscoder, msg.sender);
```

Comment:
Yeah so since an imported library can be considered an implicit base contract, if the library emits an event it will show up in the main contract's event log. ATM, the main contract's ABI will not reflect the events of the main contract's imported libraries so clients will not be able to decode events emitted from libraries. However, if you define an event `A` in the main contract with the same signature as the event `A` defined in the library, then the main contract ABI will reflect that event `A` and clients will be able to decode that event `A`.